### ISSUE TITLE
[Accessibility] Close button presents in chip is not accessible via keyboard

### ISSUE THREAD

**Issue by opozo** [angular/components#11896]
#### Bug, feature request, or proposal:

Chip close button is not accessible [1].
[1] https://material.angular.io/components/chips

#### What is the expected behavior?

Chip close button should be reached using keyboard.

#### What is the current behavior?

Close button presents in chip is not reached via keyboard.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?

@angular/material: 6.3.0

**Comment by opozo** [comment]
#

**Comment by EverettZ** [comment]
According to [#11954](https://github.com/angular/material2/pull/11954#issuecomment-400767251) it is intended to be this way. Should this issue be closed?

**Comment by jelbourn** [comment]
Duplicate of #15005
